### PR TITLE
Added php 7.2 with sqlite to the matrix

### PR DESCRIPTION
**Pull request by SergioBertolinSG**
Still not ready for php 7.2 . It fails showing the warning about running with 7.2:

```
This version of ownCloud is not compatible with PHP 7.2
```

### CODE REVIEW

**Comment by SergioBertolinSG** [comment]
I think it is missing a fix for this commit :

 https://github.com/owncloud/core/pull/28575/commits/dc77daf44036464e728884ce3ef33994d471dc0c

**Comment by DeepDiver1975** [comment]
This needs to be adjusted as well - yes - @SergioBertolinSG can you take care please? THX

File: console.php [hunk 1]
```diff
@@ -41,13 +41,6 @@
 	return;
 }
 
-// Show warning if PHP 7.2 is used as ownCloud is not compatible with PHP 7.2
-if (version_compare(PHP_VERSION, '7.2.0alpha1') !== -1) {
```
Comment (DeepDiver1975) [inline console.php]: please keep this and adjust the version to 7.3

Comment (DeepDiver1975) [reply console.php]: see https://github.com/php/php-src/blob/16eb3873b0c85e1e71ecb89276cbdf03492ada78/NEWS#L3 
7.3.0alpha1 seems to be a good string to test against.

**Review by DeepDiver1975 — CHANGES_REQUESTED** [review]

File: console.php (hunk 1 above)
Comment (SergioBertolinSG) [reply console.php]: Changed.

**Comment by DeepDiver1975** [comment]
last error to be addressed:

````

 There was 1 error:

1) Test\User\BasicAuthModuleTest::testAuth with data set "user2 is not known" (false, 'user2')
count(): Parameter must be an array or an object that implements Countable

/drone/src/lib/private/User/BasicAuthModule.php:55
/drone/src/tests/lib/User/BasicAuthModuleTest.php:76

**Comment by DeepDiver1975** [comment]
jenkins failure is related to 739351e

````
12:55:28 [Tue Dec 12 12:55:28 2017] PHP Stack trace:
12:55:28 [Tue Dec 12 12:55:28 2017] PHP   1. {main}() /var/lib/jenkins/workspace/owncloud-core_core_PR-29811-RHFDMLDZR3TSA3EASJHEF2JWMTHMG3AZI3VFEETPJ3X4QEB4GTRQ@3/index.php:0
12:55:28 [Tue Dec 12 12:55:28 2017] PHP   2. OC::handleRequest() /var/lib/jenkins/workspace/owncloud-core_core_PR-29811-RHFDMLDZR3TSA3EASJHEF2JWMTHMG3AZI3VFEETPJ3X4QEB4GTRQ@3/index.php:56
12:55:28 [Tue Dec 12 12:55:28 2017] PHP   3. OC\Route\Router->match() /var/lib/jenkins/workspace/owncloud-core_core_PR-29811-RHFDMLDZR3TSA3EASJHEF2JWMTHMG3AZI3VFEETPJ3X4QEB4GTRQ@3/lib/base.php:931
12:55:28 [Tue Dec 12 12:55:28 2017] PHP   4. call_user_func:{/var/lib/jenkins/workspace/owncloud-core_core_PR-29811-RHFDMLDZR3TSA3EASJHEF2JWMTHMG3AZI3VFEETPJ3X4QEB4GTRQ@3/lib/private/Route/Router.php:342}() /var/lib/jenkins/workspace/owncloud-core_core_PR-29811-RHFDMLDZR3TSA3EASJHEF2JWMTHMG3AZI3VFEETPJ3X4QEB4GTRQ@3/lib/private/Route/Router.php:342
12:55:28 [Tue Dec 12 12:55:28 2017] PHP   5. OC\Route\Route->OC\Route\{closure}() /var/lib/jenkins/workspace/owncloud-core_core_PR-29811-RHFDMLDZR3TSA3EASJHEF2JWMTHMG3AZI3VFEETPJ3X4QEB4GTRQ@3/lib/private/Route/Router.php:342
12:55:28 [Tue Dec 12 12:55:28 2017] ::1:59860 [500]: /index.php/apps/files_trashbin/ajax/delete.php - require_once(): Failed opening required '..' (include_path='/var/lib/jenkins/workspace/owncloud-core_core_PR-29811-RHFDMLDZR3TSA3EASJHEF2JWMTHMG3AZI3VFEETPJ3X4QEB4GTRQ@3/apps') in /var/lib/jenkins/workspace/owncloud-core_core_PR-29811-RHFDMLDZR3TSA3EASJHEF2JWMTHMG3AZI3VFEETPJ3X4QEB4GTRQ@3/lib/private/Route/Route.php on line 155
12:55:28 [Tue Dec 12 12:55:28 2017] require_once(): Failed opening required '..' (include_path='/var/lib/jenkins/workspace/owncloud-core_core_PR-29811-RHFDMLDZR3TSA3EASJHEF2JWMTHMG3AZI3VFEETPJ3X4QEB4GTRQ@3/apps') at /var/lib/jenkins/workspace/owncloud-core_core_PR-29811-RHFDMLDZR3TSA3EASJHEF2JWMTHMG3AZI3VFEETPJ3X4QEB4GTRQ@3/lib/private/Route/Route.php#155

**Comment by DeepDiver1975** [comment]
@SergioBertolinSG there is some error

bildschirmfoto von 2017-12-12 14-42-36

**Review by DeepDiver1975 — APPROVED** [review]

File: index.php [hunk 2]
```diff
@@ -35,9 +35,9 @@
 	return;
 }
 
-// Show warning if PHP 7.2 is used as ownCloud is not compatible with PHP 7.2
-if (version_compare(PHP_VERSION, '7.2.0alpha1') !== -1) {
-	echo 'This version of ownCloud is not compatible with PHP 7.2<br/>';
+// Show warning if PHP 7.3 is used as ownCloud is not compatible with PHP 7.3
+if (version_compare(PHP_VERSION, '7.3.0alpha1') !== -1) {
+	echo 'This version of ownCloud is not compatible with PHP 7.3<br/>';
 	echo 'You are currently running ' . PHP_VERSION . '.';
```
Comment (DeepDiver1975) [inline index.php]: please be more careful when resolving conflicts - the recently introduced changes just got killed - see https://github.com/owncloud/core/blob/877715561fdac5dff46d98c26791c180eecf1a3c/index.php#L32-L41 @SergioBertolinSG

Comment (DeepDiver1975) [reply index.php]: same in console.php

**Review by DeepDiver1975 — CHANGES_REQUESTED** [review]

**Comment by DeepDiver1975** [comment]
conflicting @SergioBertolinSG

**Comment by DeepDiver1975** [comment]
1149c05f2c5d2fe1c3781a5574bcc54d5ca53530 needs to be squashed into 8806702b3f0885f8d1075922178a1f8e6c83ff81 - this is where the trouble started @SergioBertolinSG

**Review by DeepDiver1975 — APPROVED** [review]

**Pull request merged**